app/jobs/regular/topic_reminder.rb

```
@@ -0,0 +1,28 @@
+module Jobs
+  class TopicReminder < Jobs::Base
+
+    def execute(args)
+      topic_status_update = TopicStatusUpdate.find_by(id: args[:topic_status_update_id])
+
+      topic = topic_status_update&.topic
+      user = topic_status_update&.user
+
+      if topic_status_update.blank? || topic.blank? || user.blank? ||
+          topic_status_update.execute_at > Time.zone.now
+        return
+      end
+
+      user.notifications.create(
```

I usually use `create!`  so that it doesn't silently fail.